scripts/process_schemas.py

```diff
@@ -266,7 +288,10 @@ def writeHeader(self, outputFile):
         print(file=outputFile)
         print("import avro.schema", file=outputFile)
         print(file=outputFile)
-        versionStr = self.version[1:]  # Strip off leading 'v'
+        if self.version[2] == '.':
+            versionStr = self.version[1:]  # Strip off leading 'v'
+        else:
+            versionStr = self.version
         print("version = '{0}'".format(versionStr), file=outputFile)
```

Comment:
+1 @saupchurch, not pedantic at all, on the contrary a very valid, fail-safe point.